WORKSPACE

```diff
@@ -349,6 +349,15 @@ container_pull(
     repository = "kubevirtci/nfs-ganesha",  # see https://github.com/slintes/docker-nfs-ganesha
 )
 
+# Pull fedora container-disk preconfigured with ci tooling
+# like stress and qemu guest agent pre-configured
+container_pull(
+    name = "fedora_with_test_tooling",
+    digest = "sha256:555955be79c502809002fb3af31b0d702b85c38ee306c1dd6d2d6b7bc20e633a",
+    registry = "quay.io",
+    repository = "dvossel/fedora-with-test-tooling",
```

Comment:
Ok, we need a job which pushes this to `quay.io/kubevirtci/fedora-with-test-tooling`.

Reply:
:+1:  yup, good catch, i forgot this.

Reply:
done